README.md

```diff
@@ -438,6 +438,12 @@ Note: the arguments passed to `bazel run` after `--` are forwarded to the execut
 
 [bazel instructions]: https://docs.bazel.build/versions/master/install.html
 
+### Toolchains
+
+Aka cross-compilation aka building docker images on mac/windows. When you add `node_repositories()` to your `WORKSPAE` file it will setup node for all currently supported platforms, Linux, macOS and Windows. If you want to build an app or a docker image for another platform you have to pass in the `--platforms` flag to Bazel. E.g. `bazel build --platforms=@build_bazel_rules_nodejs//toolchains/node:linux_amd64 //app` will ensure that the linux nodejs binary is downloaded and provided to the nodejs_binary.
```

Comment:
nit: spelling `WORKSPACE`

Maybe put the bit about cross compilation & building docker images for another platform to a subheading such as #### Cross-compilation.

Also, I don't think "aka" quite fits as toolchains serve another purpose which is to de-couple the nodejs toolchain definition from the usage in nodejs_binary (https://docs.bazel.build/versions/master/toolchains.html#overview). This also enables RBE when building on osx & windows. A better description could be that toolchains "allow" cross compilation and building docker images for non-host platforms?